src/renderer/components/Settings.tsx

```diff
@@ -34,6 +34,7 @@ export default function SettingsUi() {
             true,
             () => Settings.tray ?? true
         ],
+        !isMac && ["middleClickAutoscroll", "Middle Click Autoscroll", "Middle-clicking scrolls instead of pasting content from PRIMARY (Requires a full restart)", false],
```

Comment:
```suggestion
        isLinux && ["middleClickAutoscroll", "Middle Click Autoscroll", "Middle-clicking scrolls instead of pasting content from PRIMARY (Requires a full restart)", false],
```

Reply:
this message is also misleading. it still pastes anyway

Reply:
Implemented the fixes you suggested and updated the description to be less misleading